packages/newspringchurchapp/src/tabs/home/Features/index.js

```diff
@@ -103,9 +103,7 @@ const Features = memo(({ navigation }) => (
                 }
                 actions={actions.map((action) => ({
                   ...action,
-                  subtitle: action.subtitle
-                    ? action.subtitle.split(' - ').pop()
-                    : '',
+                  subtitle: '',
```

Comment:
@redreceipt I'll admit, this feels like a hack. But again, I'd have to override a function we haven't overridden yet to make this happen API side. What do you think?

Reply:
you could instead just not ask for the subtitle in the query and then we could prob match that line to core

Reply:
That's a good call. Looks better that way.